app/presenters/reservations/calendar_presenter.rb

```diff
@@ -23,6 +23,12 @@ def reservation_options(options)
           title: "Instrument Offline",
           email: created_by.try(:full_name),
         }
+      elsif expires_mins_before.present?
```

Comment:
It's not bad, but I might do something like

```
else
  hash = {
    title: model_name.human,
    email: created_by.try(:full_name),
  }
  hash[:expiration] = "Expires #{display_as_time(expires_mins_before)} prior" if expires_mins_before.present?
  hash
```

We don't necessarily need to do it, but what would happen if we changed `OfflineReservations`'s model name to "Instrument Offline" and collapse even another conditional?

Reply:
I don't entirely know all the spots where we use OfflineReservation's model name so am a little hesitant to do that now, but you make good sense.